ts/test/session/unit/libsession_wrapper/libsession_wrapper_contacts_test.ts

```diff
@@ -137,6 +137,45 @@ describe('libsession_contacts', () => {
       ).to.be.eq(false);
     });
 
+    it('excludes contacts not matching a pubkey syntax (space in middle)', () => {
+      const validIdWithSpaceInIt =
+        '050123456789abcdef050123456789 bcdef0123456789abcdef050123456789ab'; // len 66 but has a ' ' in the middle
+      expect(
+        SessionUtilContact.isContactToStoreInWrapper(
+          new ConversationModel({
+            ...validArgs,
+            id: validIdWithSpaceInIt,
+          } as any)
+        )
+      ).to.be.eq(false);
+    });
+
+    it('excludes contacts not matching a pubkey syntax (space at the end)', () => {
+      const validIdWithSpaceInIt =
+        '050123456789abcdef050123456789abcdef0123456789abcdef050123456789a '; // len 66 but has a ' ' at the end
+      expect(
+        SessionUtilContact.isContactToStoreInWrapper(
+          new ConversationModel({
+            ...validArgs,
+            id: validIdWithSpaceInIt,
+          } as any)
+        )
+      ).to.be.eq(false);
+    });
```

Comment:
What about space at the beginning? Although I don't think it will make a difference